Assets/GothicVR/Scripts/Player/Menu/MenuManager.cs

```diff
@@ -0,0 +1,87 @@
+using GVR.Creator;
+using GVR.Manager;
+using GVR.Phoenix.Util;
+using UnityEngine;
+
+namespace GVR.Player.Menu
+{
+    public class MenuManager : MonoBehaviour
+    {
+        public GameObject MainMenu;
+        public GameObject SettingsMenu;
+        public GameObject TeleportMenu;
+        public GameObject MovementMenu;
+        public GameObject UIMenu;
+
+        void Awake()
+        {
+            SetSettingsValues();
+        }
+
+        public void SetSettingsValues()
+        {
+            var moveSetting = MovementMenu.FindChildRecursively("MoveSpeedSlider").GetComponent<MoveSpeedController>();
+            moveSetting.ChangeMoveSpeed(PlayerPrefs.GetFloat(ConstantsManager.I.moveSpeedPlayerPref));
+
+            var turnSetting = MovementMenu.FindChildRecursively("TurnSettingDropdown").GetComponent<TurnSettingDropdownController>();
+            turnSetting.DropdownItemSelected(PlayerPrefs.GetInt(ConstantsManager.I.turnSettingPlayerPref));
+        }
+
+        public void PlayFunction()
```

Comment:
Daedalus seems to have actions defined for the menu entries. How about leveraging these and call our methods the same or somethow "map" the information of "which function to call"?
![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/ced37927-702e-42a0-8c74-63559c0fea2b)

Reply:
not using c-menu from daedalus